src/client/containers/404Page/404Page.styles.css

```diff
@@ -1,3 +1,17 @@
-div {
-  color: #687808;
+.page404 {
+  text-align: center;
+}
```

Comment:
Here I whould add some margin top and bottom to make it look more like figma.

Reply:
Thank you .